ui/changes/src/index.js

```diff
@@ -334,13 +316,154 @@ function renderTestingSummary(bugSummaries) {
     },
   };
 
-  let chartEl = document.createElement("div");
-  resultSummary.append(chartEl);
-
   var chart = new ApexCharts(chartEl, options);
   chart.render();
 }
 
+function renderRiskChart(chartEl, bugSummaries) {
+  let minDate = Temporal.PlainDate.from(
+    bugSummaries.reduce((minSummary, summary) =>
```

Comment:
We need to handle empty array somehow (see screenshot of js error earlier). Maybe just early return at the start if bugSummaries.length is 0.